modules/replicator/src/main/java/org/apache/ignite/internal/replicator/Replica.java

```diff
@@ -68,7 +68,7 @@ public class Replica {
     public static final ReplicaResult EMPTY_REPLICA_RESULT = new ReplicaResult(null, null);
 
     /** Replica group identity, this id is the same as the considered partition's id. */
-    private final ReplicationGroupId replicaGrpId;
+    private final ZonePartitionId replicaGrpId;
```

Comment:
Zone partition id already existed here.
Remove the duplicate.

Reply:
done